valve_ds/fast_valve.py

```diff
@@ -0,0 +1,81 @@
+"""
+This module contains device class Valve and run method for it.
+"""
+
+# Imports
+from tango import DevState, AttrWriteType, DispLevel
+from facadedevice import proxy_attribute, state_attribute
+from valve import Valve
+
+
+class FastValve(Valve):
+    """
+
+    InrushA1 should be True if air-inrush is detected on first sensor or False
+    in normal state.
+    InrushA2 should be True if air-inrush is detected on second sensor or False
+    in normal state.
+    """
+    __doc__ = Valve.__doc__ + __doc__
```

Comment:
Yes, it does, however only with class docstring. I've tried that with `state_and_status_fast_valve` method docstring and it hasn't worked out.